packages/cli/medusa-cli/src/create-cli.ts

```diff
@@ -125,6 +125,28 @@ function buildLocalCommands(cli, isLocalProject) {
       desc: `Create a new Medusa project.`,
       handler: handlerP(newStarter),
     })
+    .command({
+      command: `db:create`,
+      desc: `Manage migrations from the core and your own project`,
+      builder: (builder) => {
+        builder.option("db", {
+          type: "string",
+          describe: "Specify the name of the database you want to create",
+        })
+        builder.option("interactive", {
+          type: "boolean",
+          default: true,
+          describe:
+            "Skip prompts and use the default value for the database name",
```

Comment:
suggestion: I feel like the description should be the otherway around wdyt?

Reply:
Since the default value for this prompt is `true`. You will instead use `--no-interactive` to hide the prompt.

Reply:
But if you show the help it will show you interactive skip the prompts no?

Reply:
What do you think of `Display prompts. Use --no-interactive flag to run the command without prompts`

Reply:
I think it would be nicer 👍